src/components/ActionBar.vue

```diff
@@ -907,15 +906,10 @@ div.bar-filter-wrapper
         q-card-actions(align="right" class="bg-white")
           q-btn(flat label="OK" v-close-popup)
 </template>
-<style lang="sass">
+<style lang="sass" scoped>
```

Comment:
I think we talked about using BEM and thus, not using scoped css 🤔

Reply:
I'm aware of what you say and I agree. However, this component (ActionBar) has its own style impacting the whole project because is not written using any standard or strategy to avoid it.
So I had to setted it as scoped on purpose to avoid that spreading effect.

But I agree with you that this should not be scoped. So I created issue #217 to fix this problem.